Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
-* extension.valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* dosage.text = "250mg orally once daily with or without food"
+* dosage.route = SCT#26643006 "Oral use"
+* dosage.dose[x].system = UCUM#mg "mg"
+* dosage.dose[x].value = "250"
 
 
 Instance: mCODECancerRelatedSurgicalProcedureExample01
 InstanceOf: CancerRelatedSurgicalProcedure
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedSurgicalProcedure"
 * status = #completed "completed"
+* category = SCT#387713003 "Surgical procedure"
 * code = SCT#359615001 "Partial lobectomy of lung (procedure)"
 * subject = Reference(mCODEPatientExample01)
 * asserter = Reference(mCODEPractitionerExample01)
 * performedDateTime = "2019-03-01"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * reasonReference = Reference(mCODEPrimaryCancerConditionExample01)
 * bodySite = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
+
+
+Instance: mCODECancerRelatedRadiationProcedureExample01
+InstanceOf: CancerRelatedRadiationProcedure 
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedRadiationProcedure"
+* status = #completed "completed" 
+* category = SCT#53438000 "Radiation therapy procedure or service (procedure)"
+* code = SCT#152198000 "Brachytherapy (procedure)"
+* subject = Reference(mCODEPatientExample01)
+* asserter = Reference(mCODEPractitionerExample01)
+* performedDateTime = "2019-03-01"
+* extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.code = UCUM#cGy 
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.value = "1200"
+* reasonReference = Reference(mCODEPrimaryCancerConditionExample01)
+* bodySite = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
+
+
+Instance: mCODETNMClinicalDistantMetastasesCategoryExample01
+InstanceOf: TNMClinicalDistantMetastasesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalDistantMetastasesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21907-1 "Distant metastases.clinical [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#cM0 "M0"
+
+
+Instance: mCODETNMClinicalPrimaryTumorCategoryExample01
+InstanceOf: TNMClinicalPrimaryTumorCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalPrimaryTumorCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21905-5 "Primary tumor.clinical [Class] Cancer"
```

Comment:
inherited, I think...

Reply:
Clinical staging is not a lab, but Pathologic staging is a lab. Double check all the examples.

Reply:
That's my bad. It was in the CIMPL examples that way and likely a typo from copying it from the pathologic staging examples.  
@CarolinePotteiger - change it to
`* category = #survey "survey"`
Just for the clinical staging examples.
As noted, pathologic staging category remains as lab.